HTTP2-Core.package/H2Stream.class/instance/headerCompleted.st

```diff
@@ -1,5 +1,9 @@
 responding
 headerCompleted
-	self responseHeader: 
-		(H2Header newFrom: self byteStreamIn contents in: self connection headerTableIn).
-	self byteStreamIn resetContents
\ No newline at end of file
+	| receivedHeader |
+	
+	receivedHeader := (H2Header newFrom: self headerStreamIn contents in: self connection headerTableIn).
+	self headerStreamIn resetContents.
+	receivedHeader dictionary at: ':status' ifAbsent: [ ^self ]. "malformed request -> RFC 7540 8.1.2.4."
+	((receivedHeader at: ':status') < '200') ifTrue: [ ^self ]. "100 and 102 don't need handling, 101 does"
```

Comment:
line 7/8 - should it be receivedHeader at: or ... dictionary at: ?

Reply:
H2Header doesn't have ```at:ifAbsent```, only ```at:``` as a proxy for its dictionary.
I guess adding ```at:ifAbsent:``` to H2Header would be better. I'll add it.